### ISSUE TITLE
Support search functionality in DLNA browser

### ISSUE THREAD

**Issue by szotsaki** [jellyfin/jellyfin-plugin-dlna#41]
Music DLNA browser screen offers many groupings (newest, playlists, albums, songs, favourite artists, etc.) but if you have more than 40-50 different songs you will need the search functionality. More if you have more than 1000+ songs.

Please, add a new section for searching inside the Jellyfin music (and perhaps movies) database.

DLNA browser is many times a must for built-in devices (e.g. an AV receiver with its own smartphone app), therefore this interface cannot be substituted by the Web UI (e.g. Jellyfin app).

A similar thread I found: https://news.ycombinator.com/item?id=30923571

**Comment by disgustipated** [comment]
I tried the search functionality on my TV and it showed results from my server and it seemed to work. Does the device youre using have capabilities for search?

**Comment by opapanik** [comment]
Hey @disgustipated don't know what your TV did (maybe you're talking about jellyfin?), but this DLNA plugin (even when it was not a plugin) does not support **any** search functionality. 

Really makes jellyfin unusable as a DLNA server when hundreds or thousands of items are involved. And that's a pity considering so much work has gone into it.

It is really something that's missing for a long time. Please add DLNA search capabilities. I am using jellyfin also for music, and on DLNA only devices would like to see at least artist, album and track DLNA search.

Thank you

**Comment by disgustipated** [comment]
> (maybe you're talking about jellyfin?)

No, not the web os jellyfin app. i click the search option in the dlna browser on the tv. i search for an item and the criteria that matches displays. are you getting an error or something?

**Comment by szotsaki** [comment]
Unfortunately, I don't see any search functionality on my AV receiver browser either. Because of https://github.com/jellyfin/jellyfin-plugin-dlna/issues/48 I can't try it right now, but if DLNA works for you, could you please try [YAACC](https://f-droid.org/packages/de.yaacc/)?

**Comment by Shadowghost** [comment]
Fixed on unstable
